twilio/twiml/messaging_response.py

```diff
@@ -30,7 +30,7 @@ def message(self,
         :param action: action URL
         :param status_callback: callback URL
         :param kwargs: other attributes
-        :return: <Message> element
+        :return: <MessagingResponse> element
```

Comment:
`self.append` returns the parent element, in this case `MessagingResponse` or a `<Response>` element.